scripts/templates/detector.tf.j2

```diff
@@ -32,9 +32,9 @@ resource "signalfx_detector" "{{ id }}" {
 
     {%- for key, signal in signals.items() -%}
       {%- if 'metric' in signal %}
-    {{ key }} = data('{{ signal.metric }}', filter={%- if exclude_not_running_vm is defined and exclude_not_running_vm -%}${local.not_running_vm_filters} and {% endif -%}{%- if filtering is defined and filtering -%}{{ filtering_variable }} and {% endif -%}${module.filter-tags.filter_custom}{%- if signal.extrapolation is defined and signal.extrapolation -%}, extrapolation='{{ signal.extrapolation }}'{% endif -%})
+    {{ key }} = data('{{ signal.metric }}', filter={%- if exclude_not_running_vm is defined and exclude_not_running_vm -%}${local.not_running_vm_filters} and {% endif -%}{%- if filtering is defined and filtering -%}{{ filtering_variable }} and {% endif -%}${module.filter-tags.filter_custom}{%- if signal.extrapolation is defined and signal.extrapolation -%}, extrapolation='{{ signal.extrapolation }}'{% endif -%}{%- if signal.rollup is defined -%}, rtype='{{ signal.rollup }}'{% endif -%})
```

Comment:
`rtype=` -> `rollup=` ?

we already use `rollup` in this repo and we use `filter` (not `fobject`) so it will be more consistent and understandable.

Reply:
also it will be better to add the rollup *before* the extrapolation as the signalfx builder does (it will avoid undesirable diff after edit from webui)